src/modules/auth/controller.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const jwt = require('jsonwebtoken');
+const { validateAuth } = require('./validators');
+const validator = require('../../middlewares/validator');
+const passwordService = require('../password/service');
+const userService = require('../user/service');
+const {
+  SECRET,
+  INVALID_CREDENTIALS_MESSAGE,
+  TOKEN_EXPIRATION_TIME,
+} = require('./constants');
+const { StatusCodes } = require('http-status-codes');
+const { UnauthorizedError } = require('../../utils/errors');
+
+const router = express.Router();
+
+const tryAuth = async (req, res, next) => {
+  const { email, password } = req.body;
+
+  try {
+    const passwordHash = await userService.getUserPasswordHash(email);
+    if (passwordHash && (await passwordService.compare(password, passwordHash))) {
```

Comment:
Clean it up
1. Change `getUserPasswordHash` to `getUser` it will be more extendable
2. You can throw an error inside passwordService.compare method in case if user object is undefined/null or in case if passwords dont match -- this will help you to avoid extra `else` statement in controller and also `if` statement will be more simple